f/medical/fn_famInit.sqf

```diff
@@ -0,0 +1,68 @@
+// F3 FA Medical
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+// ====================================================================================
+
+// INITIALIZE
+_unit = _this;
+
+// ====================================================================================
+
+// MEDICAL SCRIPTS
+// Start Health Loop for monitoring bleed/consciouness
+_unit spawn f_fnc_famLoop;
+
+// Make downed players draggable
+[player] remoteExec ["f_fnc_famAddDragAction", 0, player]; 
+
+// Handle damage appropriately for the system.
+_unit call f_fnc_famDamageHandler;
+
+// ====================================================================================
+
+// MEDICAL VARIABLES
+// These become global later.
+_unit setVariable ["FAM_BLEED",false]; 
+_unit setVariable ["FAM_CONSCIOUS",true]; 
+
+// These are local only.
+_unit setVariable ["FAM_FORCEDOWN",false]; 
+_unit setVariable ["FAM_HASFAK",false]; 
+_unit setVariable ["FAM_HASBANDAGE",false]; 
+_unit getVariable ["FAM_FLAG",false];
+FAM_MEDICMOD = 0.5;
```

Comment:
Instead of `FAM_*` we could think about using our (otherwise consistent?) `f_var_*` naming scheme for global variables to avoid conflicts with mission maker variables